### DISCUSSION TITLE
Get Galaxy Brain badge

### DISCUSSION THREAD

**Discussion by callqh** [Schweinepriester/github-profile-achievements#432] [Q&A]
Get Galaxy Brain badge！

**Comment by callqh** [comment]
👀

**Comment by TsaiRongFu** [comment]
@liuqh0609 answer！

**Comment by DevPro15** [comment]
> Get Galaxy Brain badge！

Your answer should be accepted 2 time to achieve the brain badge
try to answer other problems and participate in discussions

i hope it will help you
